admin/src/css/variables.less

```diff
@@ -0,0 +1,2 @@
+@warning-color: #C78330;
+@error-color: #DA4548;
```

Comment:
Can you import the `variables.less` from webapp? This will make reuse easier and keep the app styles more consistent.